infrastructure/infrastructure/src/main/java/alabno/utils/ConnUtils.java

```diff
@@ -9,11 +9,31 @@
  */
 public class ConnUtils {
 
+    @SuppressWarnings("unchecked")
     public static void sendAlert(WebSocket conn, String string) {
         JSONObject msgobj = new JSONObject();
         msgobj.put("type", "alert");
         msgobj.put("message", string);
         conn.send(msgobj.toJSONString());
     }
     
+    public enum Color {
+        RED, GREEN, YELLOW, BLACK;
```

Comment:
I would debate that this is more of a implementation detail for the frontend client and could change (we might not even use colors to show this in the end), so it's a bit fragile. It may be more useful to give what the actual status of the job is and let the representation shown to the user decide what to do / show with these statuses.